### ISSUE TITLE
Find out why the build pipeline is not using the right repo

### ISSUE THREAD

**Issue by sebohe** [hashcloak/Meson#88]

**Comment by sebohe** [comment]
I have forks of katzenpost/server and katzenpost/core that are not being used in the pipeline.

**Comment by sebohe** [comment]
The correct repository are being used... but the built images don't include the WarpedEpoch build time library.

**Comment by sebohe** [comment]
This has been an odd bug to debug, and I can't fully explain what is happening because I still don't understand how it does it.

The way Meson-plugin uses the Docker containers is that we make a `hashcloak/server` based on https://github.com/katzenpost/server. Since Meson is a plugin what we do is that we create a new container called `hashcloak/meson` that is continuation of the `hashcloak/server` container but we pop into the container the plugin binary. The server binary inside the container is built in the `hashcloak/server` build step.

It is important to note that the server binary is what has the `WarpedEpoch` flag set, so any changes done to the plugin binary should not affect the warped epoch.

But I ended up in a situation in which the tag name of `hashcloak/meson` somehow is affecting the `WarpedEpoch` flag of the server. The current solution is to remove any underscore and dash: https://github.com/hashcloak/Meson-plugin/commit/dc8562b1bab58ef11ee5172ed0efcae61e53ebb3#diff-2b9c94f5cd8d9de2e8423415651bae4aR34

There are two possibilities:

1. Either the build scripts are wrong and using the wrong container (I have checked for this step by step)
2. A docker bug.

It is most likely that option 1 is the cause.

Closing this issue for now.
